.github/workflows/wrappers.yml

```diff
@@ -36,607 +29,618 @@ jobs:
-      if: matrix.os == 'windows'
+    - if: matrix.os == 'windows'
+      run: git clean -fdx
+    - name: Check Cache for iOS-Device
+      id: check-cache-iOS-Device
+      if: matrix.os == 'ubuntu'
+      uses: actions/cache/restore@v4
+      with:
+        path: ./wrappers/build/**
+        key: wrappers-iOS-Device-Release-${{hashFiles('./wrappers/**')}}
+    - name: Store artifacts for wrappers-iOS-Device
+      if: matrix.os == 'ubuntu' && steps.check-cache-iOS-Device.outputs.cache-hit == 'true'
+      uses: actions/upload-artifact@v4
+      with:
+        name: wrappers-iOS-Device
+        path: wrappers/build/**
+        retention-days: ${{ github.event_name != 'pull_request' && 30 || 1 }}
+        if-no-files-found: error
+    - if: matrix.os == 'ubuntu'
+      run: git clean -fdx
+    - name: Check Cache for iOS-Simulator
+      id: check-cache-iOS-Simulator
+      if: matrix.os == 'ubuntu'
+      uses: actions/cache/restore@v4
+      with:
+        path: ./wrappers/build/**
+        key: wrappers-iOS-Simulator-Release-${{hashFiles('./wrappers/**')}}
+    - name: Store artifacts for wrappers-iOS-Simulator
+      if: matrix.os == 'ubuntu' && steps.check-cache-iOS-Simulator.outputs.cache-hit == 'true'
+      uses: actions/upload-artifact@v4
+      with:
+        name: wrappers-iOS-Simulator
+        path: wrappers/build/**
+        retention-days: ${{ github.event_name != 'pull_request' && 30 || 1 }}
+        if-no-files-found: error
+    - if: matrix.os == 'ubuntu'
+      run: git clean -fdx
+    - name: Check Cache for tvOS-Device
+      id: check-cache-tvOS-Device
+      if: matrix.os == 'ubuntu'
+      uses: actions/cache/restore@v4
+      with:
+        path: ./wrappers/build/**
+        key: wrappers-tvOS-Device-Release-${{hashFiles('./wrappers/**')}}
+    - name: Store artifacts for wrappers-tvOS-Device
+      if: matrix.os == 'ubuntu' && steps.check-cache-tvOS-Device.outputs.cache-hit == 'true'
+      uses: actions/upload-artifact@v4
+      with:
+        name: wrappers-tvOS-Device
+        path: wrappers/build/**
+        retention-days: ${{ github.event_name != 'pull_request' && 30 || 1 }}
+        if-no-files-found: error
+    - if: matrix.os == 'ubuntu'
+      run: git clean -fdx
+    - name: Check Cache for tvOS-Simulator
+      id: check-cache-tvOS-Simulator
+      if: matrix.os == 'ubuntu'
+      uses: actions/cache/restore@v4
+      with:
+        path: ./wrappers/build/**
+        key: wrappers-tvOS-Simulator-Release-${{hashFiles('./wrappers/**')}}
+    - name: Store artifacts for wrappers-tvOS-Simulator
+      if: matrix.os == 'ubuntu' && steps.check-cache-tvOS-Simulator.outputs.cache-hit == 'true'
+      uses: actions/upload-artifact@v4
+      with:
+        name: wrappers-tvOS-Simulator
+        path: wrappers/build/**
+        retention-days: ${{ github.event_name != 'pull_request' && 30 || 1 }}
+        if-no-files-found: error
+    - if: matrix.os == 'ubuntu'
+      run: git clean -fdx
+    strategy:
```

Comment:
Is there a way of putting this before all the other steps as it was before? I thought this was missing at a first glance